javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+
+var intervalText = document.querySelector('.countdown-display');
+var counter = 4;
+
+var theCountdown = function () {
+  intervalText.textContent = counter;
+  counter--;
+  if (counter < 0) {
+    intervalText.textContent = '~Earth Beeeeelooowww Ussss~';
+    clearInterval(theCountdown);
```

Comment:
`theCountdown` is not actually holding the interval id, this line is doing nothing. The interval id is the return of calling `setInterval`. In your code, `theCountdown` is the function definition.